src/js/components/header/search.js

```diff
@@ -0,0 +1,38 @@
+var $ = require('jquery');
```

Comment:
Is it OK that this is jquery code? Whats the plan to merge it in with the jquery removal?